src/tests/BuildWasmApps/Wasm.Build.Tests/BuildPublishTests.cs

```diff
@@ -70,11 +70,15 @@ public void BuildThenPublishNoAOT(BuildArgs buildArgs, RunHost host, string id)
         }
 
         [Theory]
-        [BuildAndRun(host: RunHost.Chrome, aot: true, config: "Release")]
-        [BuildAndRun(host: RunHost.Chrome, aot: true, config: "Debug")]
-        public void BuildThenPublishWithAOT(BuildArgs buildArgs, RunHost host, string id)
+        [BuildAndRun(host: RunHost.Chrome, aot: true, config: "Release", testUnicode: false)]
+        [BuildAndRun(host: RunHost.Chrome, aot: true, config: "Debug", testUnicode: false)]
```

Comment:
Looks like you are missing the corresponding changes to BuildAndRunAttribute?

Reply:
With @radical we decided to introduce some changes to tests in net8 first, maybe it will help resolve https://github.com/dotnet/runtime/issues/83497 and only then backport it.